src/RecoveryServices/RecoveryServices.Backup.Models/AzureModels/AzureItem.cs

```diff
@@ -52,6 +52,21 @@ public class AzureItem : ItemBase
         /// </summary>
         public AzureItemExtendedInfo ExtendedInfo { get; set; }
 
+        /// <summary>
+        /// Date of purge for the item
+        /// </summary>
+        public DateTime? DateOfPurge { get; set; }
+
+        /// <summary>
+        /// Indicates if the item is in a softdeleted state
+        /// </summary>
+        public bool IsScheduledForPurge { get; set; }
+
+        /// <summary>
+        /// Indicates if the delete state of the item
+        /// </summary>
+        public string DeleteState { get; set; }
```

Comment:
why use string here?

Reply:
And what the difference in functionality between DeleteState and IsScheduledForPurge ?